clang/test/SemaSYCL/supported_math.cpp

```diff
@@ -26,39 +30,51 @@ int main() {
   kernel<class kernel_function>([=]() {
     int acc[1] = {5};
     acc[0] *= 2;
-    acc[0] += (int)truncf(1.0f);                     // expected-no-diagnostics
-    acc[0] += (int)trunc(1.0);                       // expected-no-diagnostics
-    acc[0] += (int)roundf(1.0f);                     // expected-no-diagnostics
-    acc[0] += (int)round(1.0);                       // expected-no-diagnostics
-    acc[0] += (int)rintf(1.0f);                      // expected-no-diagnostics
-    acc[0] += (int)rint(1.0);                        // expected-no-diagnostics
-    acc[0] += (int)nearbyintf(0.5f);                 // expected-no-diagnostics
-    acc[0] += (int)nearbyint(0.5);                   // expected-no-diagnostics
-    acc[0] += (int)floorf(0.5f);                     // expected-no-diagnostics
-    acc[0] += (int)floor(0.5);                       // expected-no-diagnostics
-    acc[0] += (int)copysignf(1.0f, -0.5f);           // expected-no-diagnostics
-    acc[0] += (int)copysign(1.0, -0.5);              // expected-no-diagnostics
-    acc[0] += (int)sinf(1.0f);                       // expected-no-diagnostics
-    acc[0] += (int)sin(1.0);                         // expected-no-diagnostics
-    acc[0] += (int)__builtin_sinf(1.0f);             // expected-no-diagnostics
-    acc[0] += (int)__builtin_sin(1.0);               // expected-no-diagnostics
-    acc[0] += (int)cosf(1.0f);                       // expected-no-diagnostics
-    acc[0] += (int)cos(1.0);                         // expected-no-diagnostics
-    acc[0] += (int)__builtin_cosf(1.0f);             // expected-no-diagnostics
-    acc[0] += (int)__builtin_cos(1.0);               // expected-no-diagnostics
-    acc[0] += (int)logf(1.0f);                       // expected-no-diagnostics
-    acc[0] += (int)log(1.0);                         // expected-no-diagnostics
-    acc[0] += (int)__builtin_truncf(1.0f);           // expected-no-diagnostics
-    acc[0] += (int)__builtin_trunc(1.0);             // expected-no-diagnostics
-    acc[0] += (int)__builtin_rintf(1.0f);            // expected-no-diagnostics
-    acc[0] += (int)__builtin_rint(1.0);              // expected-no-diagnostics
-    acc[0] += (int)__builtin_nearbyintf(0.5f);       // expected-no-diagnostics
-    acc[0] += (int)__builtin_nearbyint(0.5);         // expected-no-diagnostics
-    acc[0] += (int)__builtin_floorf(0.5f);           // expected-no-diagnostics
-    acc[0] += (int)__builtin_floor(0.5);             // expected-no-diagnostics
-    acc[0] += (int)__builtin_copysignf(1.0f, -0.5f); // expected-no-diagnostics
-    acc[0] += (int)__builtin_logf(1.0f);             // expected-no-diagnostics
-    acc[0] += (int)__builtin_log(1.0);               // expected-no-diagnostics
+    acc[0] += (int)truncf(1.0f);                        // expected-no-diagnostics
```

Comment:
expected-no-diagnostics can be placed only once
Here is an example: https://github.com/llvm/llvm-project/blob/main/clang/test/Sema/aarch64-sve-explicit-casts-fixed-size.c

Reply:
Hi, @bader 
Thanks for the reminder, just fixed it.